2024-08-28 21:31:10.471 | ERROR    | __main__:mint_StoryNFT:48 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'tx fee (1.11 ether) exceeds the configured cap (1.00 ether)'}
2024-08-28 21:36:58.777 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'replacement transaction underpriced'}
2024-08-28 21:37:31.188 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'replacement transaction underpriced'}
2024-08-28 21:38:03.285 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'tx fee (1.11 ether) exceeds the configured cap (1.00 ether)'}
2024-08-28 21:38:35.576 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'replacement transaction underpriced'}
2024-08-28 21:39:07.638 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'replacement transaction underpriced'}
2024-08-28 21:39:44.947 | ERROR    | __main__:mint_StoryNFT:45 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-Transaction-交易哈希: b'G\x88;g\xfc\n\xc5\x10\xb0O\xadT\xf2\x890\xaa\x18\x9c5*\x14Q\x021\x0c\x12\x19\xe6\xa9\xcc\x10\xb1'-交易状态: 0
2024-08-28 21:39:44.948 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：cannot access local variable 'retry' where it is not associated with a value
2024-08-28 21:40:20.969 | ERROR    | __main__:mint_StoryNFT:45 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-Transaction-交易哈希: b'\xe5$>wL\x15\xab\xe1^\xa2\xc8\xb9\xc5\x0e\xfd3\x92\xadm\xc1|\xa1(\xddG\x89\xce;\xad\xc6x\xe7'-交易状态: 0
2024-08-28 21:40:20.970 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：cannot access local variable 'retry' where it is not associated with a value
2024-08-28 21:42:53.915 | WARNING  | tools:run_contract:91 - Error waiting for transaction receipt: Transaction HexBytes('0x02e69ffe0a3b911bd3035e7d5a5da715e1b4640fdadc73f8bfc79bdf66f202ce') is not in the chain after 120 seconds
2024-08-28 21:42:53.916 | ERROR    | __main__:mint_StoryNFT:45 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-Transaction-交易哈希: b'\x02\xe6\x9f\xfe\n;\x91\x1b\xd3\x03^}Z]\xa7\x15\xe1\xb4d\x0f\xda\xdcs\xf8\xbf\xc7\x9b\xdff\xf2\x02\xce'-交易状态: False
2024-08-28 21:42:53.917 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：cannot access local variable 'retry' where it is not associated with a value
2024-08-28 21:43:26.010 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'tx fee (1.07 ether) exceeds the configured cap (1.00 ether)'}
2024-08-28 21:43:58.171 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'replacement transaction underpriced'}
2024-08-28 21:44:30.992 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：Error in running contract function: {'code': -32000, 'message': 'replacement transaction underpriced'}
2024-08-28 21:45:06.498 | ERROR    | __main__:mint_StoryNFT:45 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-Transaction-交易哈希: b'/\xd4`\xc7\xf8\xf2$\x96\xb2\x160xU\x9d\xe9\xa33\xe2"\xbe#\x13HK\n\xc0\xcf\'S\x0f\x8aw'-交易状态: 0
2024-08-28 21:45:06.499 | ERROR    | __main__:mint_StoryNFT:49 - 0x7602280DF886111Dbb5D842EFF22622C775B978B-mint StoryNFT失败-ERROR：cannot access local variable 'retry' where it is not associated with a value
